cloudnatix/README.md

```diff
@@ -0,0 +1,50 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides the data from [CloudNatix][1].
+
+CloudNatix connects to multiple VM & K8s Clusters, enabling automated capacity, cost & operation optimization using patent-pending Autopilot technology. CloudNatix Insights also gives DevOps teams proactive visibility into potential capacity and availability issues, turbocharging productivity.
+
+The CloudNatix integration provides cost and operational optimization insights to DataDog with an out-of-the-box dashboard, allowing you to quickly view cluster cost and analyze opportunities to reduce cost.
+
+## Setup
+
+### Installation
+
+To install the CloudNatix check on your cluster:
+
+1. Build a new Docker image of Datadog agent that has the CloudNatix integration check installed.
+      * Please see [here](https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=docker) for details.
+2. [Install the Datadog Agent][3] into your Kubernetes cluster with the built Docker image.
+3. [Install CloudNatix][2] into your Kubernetes cluster. CloudNatix clusteragent will
+   automatically configure itself to work with the Datadog integration.
+
+### Validation
+
+Once set up, you will see metrics with the prefix <> appear in the [Datadog Metrics Explorer](https://app.datadoghq.com/metric/explorer). 
```

Comment:
@kenji-cloudnatix Please add the correct metric prefix above

Reply:
Updated the prefix.